### DISCUSSION TITLE
What methods can be used to Extract Bank Statement?

### DISCUSSION THREAD

**Discussion by ariefwijaya** [PaddlePaddle/PaddleOCR#12819] [Q&A]
How to extract information from this bank statement image?
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/24664153/0d34c0a0-c736-4c4a-91ac-2983d2d70dfe)

As I know, we can extract this part using KIE-SER :
<img width="769" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/24664153/67af3ac3-0184-4c83-9458-8bd3195152b9">

But, how to extract this part to get usefult result such as:
<img width="744" alt="image" src="https://github.com/PaddlePaddle/PaddleOCR/assets/24664153/dace6e66-62a1-49eb-be77-176ad177ad41">
```
[
    {
        "trx_date": "06/09",
        "trx_notes": "SALDO AWAL",
        "balance": "0.00",
        "balance_number": 0
    },
    {
        "trx_date": "06/09",
        "trx_notes": "SETORAN TUNAI",
        "trx_mutation": "50,000.00",
        "trx_mutation_number": 50000,
        "balance": "50,000.00",
        "balance_number": 50000,
        "trx_mutation_cr": "50,000.00",
        "trx_mutation_cr_number": 50000
    },
    ...
]
```

I've tried SDMGR but there is no relation between values on the same row. I'm wondering if I can use Table Recognition, but not sure. Any idea how to get it?

**Comment by ariefwijaya** [comment]
@andyjiang1116 hello, do you have idea?

**Reply by Nyaaomi** [reply]
Hi were you able to find a solution?

**Comment by CapturefastCEO** [comment]
Can you try the table detection function of https://BeyondOCR.com
And, Beyond OCR will also publish an LLM based extraction function in couple of weeks.
